ciri/api/api.c

```diff
@@ -37,22 +37,23 @@ static bool invalid_subtangle_status(iota_api_t const *const api, error_res_t **
 
 retcode_t iota_api_add_neighbors(iota_api_t const *const api, add_neighbors_req_t const *const req,
                                  add_neighbors_res_t *const res, error_res_t **const error) {
-  char **uri;
-  neighbor_t neighbor;
   retcode_t ret = RC_OK;
+  neighbor_t neighbor;
 
   if (api == NULL || req == NULL || res == NULL || error == NULL) {
     return RC_NULL_PARAM;
   }
 
   res->added_neighbors = 0;
-  for (uri = (char **)utarray_front(req->uris); uri != NULL; uri = (char **)utarray_next(req->uris, uri)) {
+
+  for (char **uri = (char **)utarray_front(req->uris); uri != NULL; uri = (char **)utarray_next(req->uris, uri)) {
```

Comment:
adding a `NEIGHBORS_FOREACH` method for readability?

Reply:
I could add it but it would be to loop over actual neighbors, these are just URIS. I can add `URIS_FOREACH` though, any idea where to put it ?

Reply:
`add_neighbors_req_t` only content URIs, so we can have NEIGHBORS_FOREACH_URI or just NEIGHBORS_FOREACH. maybe put in utils or in cclient/request/add_neighbors.h.

Reply:
Done, thanks for the suggestion !